src/components/clients/ClientForm.js

```diff
@@ -5,10 +5,19 @@ class ClientForm extends React.Component {
     super(props);
     this.state = {
       clientName: '',
+      authRedirectUrls: [],
+      authRedirectUrlsCount: 1,
+      authRedirectUrlsFocus: 0,
+      authFailureRedirectUrls: [],
+      authFailureRedirectUrlsCount: 1,
```

Comment:
so I understand the 2 "count" variables are "how many input we display". Maybe you should rename to "authRedirectUrlInputCount" and "authFailureRedirectUrlInputsCount" then.

At first read I thought it was how many urls were entered.

But maybe I'm overthinking this, so I'll let you decide.